bot/index.ts

```diff
@@ -19,24 +17,47 @@ const bot = createKaiaBotClient({
 bot.on("message", (event: MessageEvent) => {
   if (event.message.type == "text") {
     switch (event.message.text) {
-      case "hogehoge":
-        fugafuga(bot, event);
+      case "Receipt":
+        receipt(bot, event);
+        break;
+      case "Pay":
+        payment(bot, event);
         break;
       default:
-        say_hello(bot, event);
+        commandNotFound(bot, event);
     }
   }
 });
 
 bot.start();
 
-async function say_hello(bot: KaiaBotClient, event: MessageEvent) {
+async function commandNotFound(bot: KaiaBotClient, event: MessageEvent) {
   try {
     const to = event.source.userId || "";
     const messages: Array<TextMessage> = [
       {
         type: "text",
-        text: "Hello world",
+        text: "Hello. Let's start making payment",
```

Comment:
```suggestion
        text: "Hello. Let's start splitting bill.",
```

action: 
- Check receipt
- Pay to payer
- Add order